controlplane/migrations/0114_puzzling_talos.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS "namespace_config" (
```

Comment:
We should migrate the data from the old table right?

Reply:
We should, however, the way I've implemented this is, if the column for the new table doesn't have a value (is `null`), it will fallback to the value from the `namespaces` table, which we already have in memory.

Then when the customer updates any value, an `upsert` is performed with the `namespace id` as key.

Reply:
I was thinking of deleting those values from the old table after the migration is done. What do you think?

Reply:
I have reverted the migration to do copy the values to the new table and remove the old columns in a single migration instead of introducing a second one just to remove the columns